CONTRIBUTING.md

```diff
@@ -118,6 +118,75 @@ private:
 };
 ```
 
+## Translations
+For specific information on how to update and submit translations, see [here](https://wiki.wesnoth.org/WesnothTranslationsHowTo).
+
+Wesnoth has translations, at [varying levels](https://www.wesnoth.org/gettext/) of completeness, setup for the following languages:
+| Language | Maintainer | Contact |
+|----------|------------|---------|
+| [Afrikaans](https://www.wesnoth.org/gettext/?view=langs&version=branch&lang=af) | None | NA |
+| [Ancient Greek](https://www.wesnoth.org/gettext/?view=langs&version=branch&lang=grc) | None | NA |
+| [Arabic](https://www.wesnoth.org/gettext/?view=langs&version=branch&lang=ar) | None | NA |
+| [Basque](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=eu) | None | NA |
+| [Brazilian Portuguese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=pt_BR) | Andrei Machado | andreisp.machadoATyahooDOTcom |
+| [British English](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=en_GB) | Wedge009 | wedge009ATwedge009DOTnet |
+| [Bulgarian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=bg) | Ivan Petrov (TheWhiteKnight) | vankata_petrovATabvDOTbg |
+| [Burmese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=my) | None | NA |
+| [Catalan](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=ca) | Miquel-Àngel Burgos i Fradeja | miquel.angel.burgosATgmailDOTcom |
+| [Chinese](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=zh_CN) | CloudiDust | cloudidustATgmailDOTcom |
+| [Chinese (Taiwan)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=zh_TW) | 楊綮銘 | steven2880ATgmailDOTcom |
+| [Croatian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=hr) | None | NA |
+| [Czech](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=cs) | Michal Žejdl | lachimATemerDOTcz |
+| [Danish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=da) | None | NA |
+| [Dutch](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=nl) | Merijn de Vet | tybonzerATliveDOTnl |
+| [English (Shavian chars)](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=en@shaw) | NA | NA |
+| [Esperanto](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=eo) | Luther Thompson | lutherATlibremDOTone |
+| [Estonian](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=et) | None | NA |
+| [Filipino](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=tl) | None | NA |
+| [Finnish](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=fi) | Jaakko Saarikko (styxnix) | jaakkoDOTsaarikkoATprotonmailDOTcom |
+| [French](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=fr) | Mathieu Guilbaud (Guim) | NA |
+| [Galician](https://www.wesnoth.org/gettext/?view=langs&version=master&lang=gl) | Adrián Chaves Fernández (Gallaecio) | adriyetichavesATgmailDOTcom |
```

Comment:
Last active in 2018. https://forums.wesnoth.org/memberlist.php?mode=viewprofile&u=111179